src/main/java/com/clickhouse/kafka/connect/util/Utils.java

```diff
@@ -67,6 +67,7 @@ public static void handleException(Exception e) {
                 case 252: // TOO_MANY_PARTS
                 case 285: // TOO_FEW_LIVE_REPLICAS
                 case 425: // SYSTEM_ERROR
+                case 1002: // UNKNOWN_EXCEPTION
```

Comment:
It will be interesting to understand in what cases in ClickHouse it can happen, but since retry is not forever we can add this status code

Reply:
@Paultagoras, I believe it happened to you in some cases. Can you expand?

Reply:
Is it safe to assume the error is retriable? I don't think so. In what circumstances did you get it?